app/src/main/java/com/futurice/freesound/network/api/model/RequestHelper.java

```diff
@@ -20,7 +20,7 @@
 
 import android.support.annotation.NonNull;
 
-import java.util.Iterator;
+import kotlin.collections.CollectionsKt;
```

Comment:
Why didn't you just convert the file to kotlin and use `joinToString` as an extension method?